src/app/dashboard/dashboard.component.ts

```diff
@@ -1,18 +1,49 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 
 import { Project } from '../models/project';
+import { ProjectService } from '../services/project.service';
 
 @Component({
   selector: 'dashboard',
   templateUrl: './dashboard.component.html'
 })
-export class DashboardComponent {
+export class DashboardComponent implements OnInit {
 
-    // TODO: replace any type with a project model
     private projects: Project[];
+    private pageOfProjects: Project[];
+    public currentPage: number = 1;
+    public itemsPerPage: number = 5;
```

Comment:
Hm a would-be good suggestion, but with const the compiler complains, "A class member cannot have the 'const' keyword," so I'm leaving it as is.